ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3589,6 +3589,33 @@ router_l2_vpn:
     prefix_list: < prefix_list_name >
 ```
 
+### Router MSDP
+
+```yaml
+router_msdp:
+  vrfs:
+    - name: < vrf_name, if vrf_name = default, the configuration will be applied to the global section >
```

Comment:
We should not do such abstractions in eos_cli_config_gen.

Reply:
Adjusted data models/schemas/docs to match actual cli for default vrf.

Reply:
please update the description here.